### PR TITLE
[Actions] Ensure the new bot is considered.

### PR DESCRIPTION
**Pull request by mandel-macaque**
@monojenkins is no more.

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

File: .github/workflows/label-checker.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
     - run: exit 0
       name: 'Monojenkins PR'
       # always happy if monojenkins
-      if: github.actor == 'monojenkins'
+      if: github.actor == 'monojenkins' || github.actor == 'vs-mobiletools-engineering-service2'
```
Comment (spouliot) [inline .github/workflows/label-checker.yml:20]: could not `github.actor == 'monojenkins'` be removed ?

**Review by spouliot — APPROVED** [review]

File: .github/workflows/label-checker.yml (hunk 1 above)
Comment (mandel-macaque) [reply .github/workflows/label-checker.yml:20]: @spouliot could, but should I? I did think about it replacing it, I have no strong feeling

Comment (mandel-macaque) [reply .github/workflows/label-checker.yml:20]: Landing as it, but created issue: https://github.com/xamarin/xamarin-macios/issues/10359

**Pull request merged**